tests/simulation/shard_rsm.cpp

```diff
@@ -463,18 +463,18 @@ void AttemptToExpandOneWithWrongEdgeType(ShardClient &client, uint64_t src_verte
   // Edge properties to look for
   std::optional<std::vector<PropertyId>> edge_properties = {};
 
-  std::vector<msgs::Expression> expressions;
+  std::vector<std::string> expressions;
```

Comment:
Could you please add/extend a test where you use filters/expressions? Just to make sure they are working and nobody broke them in the future.